### PR TITLE
Mh/rc to play pt3

### PR DESCRIPTION
**Pull request by bleehu**
Recomend reviewing and merging pt2, which should make this much shorter.

### CODE REVIEW

**Comment by Turtlelord26** [comment]
It's so much shorter!!

**Comment by bleehu** [comment]
You're like the terminator of directors, man. You're unstoppable!

File: 01_01_Basic_Rules.txt [hunk 1]
```diff
@@ -77,20 +77,19 @@ the one-liner they want, or to have their moment of glory.
 
 	Either in combat or out, play usually follows this loop:
 
-1.	The GM describes the environment and the situation. Usually the DM will ask
-		"What do you want to do?"
+1.	The GM describes the environment and the situation. Usually the DM will ask "What do you want to do?"
```
Comment (Reveraine) [inline 01_01_Basic_Rules.txt:80]: GM vs DM inconsistancy

File: 01_01_Basic_Rules.txt [hunk 2]
```diff
@@ -77,20 +77,19 @@ the one-liner they want, or to have their moment of glory.
...
+4.	The GM says what kind of roll is needed to see if the player succeeds, if any roll is necessary.
 5.	The player rolls the dice and adds the numbers as described below.
-6. 	The GM checks their total against the Difficulty Challenge, a number that
-		typically only the GM knows which represents how hard a typical task
-		is. Then the GM narrates how the action goes depending on if the 
-		player succeeds or fails and by how much.
+6. 	The GM checks the total against the DC and narrates what happens*
 7.	Then the next player begins their turn.
 
+*The DC stands for "the Difficulty Challenge", a number that
```
Comment (Reveraine) [inline 01_01_Basic_Rules.txt:88]: "Difficulty Challenge" vs "the Difficulty Challenge"
it's DC, rather than tDC

**Review by Reveraine — APPROVED** [review]
Non-blocking grammar fixes. 
Ship it!

**Comment by bleehu** [comment]
Could we make issues to clean those two typos up? I would just do it, but I need to catch the bus.

**Review by Turtlelord26 — COMMENTED** [review]
Changes recognized by GIthub per File:

By the way don't you need this before merging according to our dev procedures?

# Basic Rules
-Style ✔️ 
-Aiming Rules, Functional change but codified houserule. Tested. ✔️ 
-Fire rate explicit rules, functional but also codified houserule. Tested. ✔️ 

# Conditions
-New condition: Disrupted. A functional change but in itself doesn't require testing. All of the testable knobs are in the abilities that reference this condition. ✔️ 

# Glossary
-Clarity ✔️ 

# Pistols | SMG & Shotguns | Heavy Weapons
-Style ✔️ 

# Armor
-Reorganized, nonfunctional ✔️ 

# Doctor Procedure
-Lethal Toxin updated to use Disrupted state, functional, untested ❌ 

# Vestigial Lv 10 weapons docs
-Deleted, nonfunctional ✔️

**Pull request merged**